### ISSUE TITLE
Miscalculation in language options

### ISSUE THREAD

**Issue by Tormentor667** [Realm667/WolfenDoom#497]
<img width="278" alt="2021-04-18_174056" src="https://user-images.githubusercontent.com/10298439/115151474-51409b00-a06d-11eb-8c6a-ab7b348d93c3.png">
I am pretty sure that english (us and uk) should never be lower than 100%.

**Comment by AFADoomer** [comment]
It's pulling that value directly from LANGUAGE...  No calculation involved in the mod.

**Comment by MuadDib90** [comment]
Interesting... 19 commits ago it was fine. I updated it just now and it shows me incomplete as to you...
So the error must be somewhere in one of the 19 commits before this last one 94f3c02 I guess.

**Comment by Username-N00b-is-not-available** [comment]
This was a Google Docs problem. It somehow appeared after a yesterday edit by @Ozymandias81 near 16:17 GMT+0, but he did not remove any English cells, so it was likely because of a faulty internet connection that the percentage cell did not update. I noticed the change in the CSV and around 19:05 GMT+0 I forced that counter to reupdate. (Technically, it was before the latest language.csv commit, but @Ozymandias81 could not have known about my edits.) The counter shows everything properly now.
Tl;dr: this is not an issue, it will be fixed the next time someone commits the most recent CSV version to the repository.
